src/Forms.js

```diff
@@ -1,149 +1,103 @@
-                    <MenuItem value={"Car loan"}>Car loan</MenuItem>
-                    <MenuItem value={"Business loan"}>Business loan</MenuItem>
-                </Select>
-                <br/>      
-                <br/>
-
-                <p>Loan amount (rub):</p>
-                <div className="slider">
-                <Slider 
-                    defaultValue={5000000}
-                    valueLabelDisplay="auto"
-                    step={500000}
-                    marks={markLoanAmount}
-                    min={0}
-                    max={10000000}
-                    onChange={handleSliderAmountChanged}
-                />
-                </div>
-                <p>Loan term (years):</p>
-                <div className="slider">
-                <Slider 
-                    defaultValue={15}
-                    valueLabelDisplay="auto"
-                    step={1}
-                    marks={markLoanTerm}
-                    min={0}
-                    max={30}
-                    onChange={handleSliderTermChanged}
-                />
-                </div>
-                <p>Choose a bank:</p>
-                <select value={bank} onChange={(event) => setBank(event.target.value)}>
-                   {banks}
-                </select>
-
-                <br/>
-
-
-                <br/>      
-                <br/>
-                <button onClick = {submit}>Submit</button>
-            </div>
-        );
+  return (
+    <Grid container spacing={2}>
+      <Grid item md={4} >
+        <h1>Requestor's data:</h1>
+        <TextField id="inputName" label="Name" variant="outlined" onChange={(event) => setName(event.target.value)} /><br /><br />
+        <TextField id="inputAge" label="Age" variant="outlined" onChange={(event) => setAge(event.target.value)} /><br /><br />
+        <TextField id="inputEmail" label="Email" variant="outlined" onChange={(event) => setEmail(event.target.value)} />
+      </Grid>
+      <Grid item md={4}>
+        <div>
+          <h1 id="greeting">Choose your credit</h1>
+          <p>Choose a loan offer:</p>
+
+          <Select value={loanOffer}
+            displayEmpty
+            onChange={(event) => setLoanOffer(event.target.value)}>
+            <MenuItem value="" disabled>Select a loan offer</MenuItem>
+            <MenuItem value={"Cash"}>Cash</MenuItem>
+            <MenuItem value={"Credit cards"}>Credit cards</MenuItem>
+            <MenuItem value={"Mortgage"}>Mortgage</MenuItem>
+            <MenuItem value={"Car loan"}>Car loan</MenuItem>
+            <MenuItem value={"Business loan"}>Business loan</MenuItem>
+          </Select>
+          <br />
+          <br />
+
+          <p>Loan amount (rub):</p>
+          <div className="slider">
+            <Slider
+              defaultValue={defaultAmountValue}
+              valueLabelDisplay="auto"
+              step={500000}
+              marks={markLoanAmount}
+              min={0}
+              max={10000000}
```

Comment:
`step` and `max` should be moved to constants as well. The same for slider down to the code

Reply:
done